### PR TITLE
Improve specfile

### PR DESCRIPTION
**Pull request by richardkiss**

### CODE REVIEW

**Review by hoffmang9 — DISMISSED** [review]
This looks good

**Comment by richardkiss** [comment]
I didn't think the `start_*` binaries (`start_wallet`, etc.) were used, but it turns out they are. The only benefit I can see to doing it this way is so they appear in `Activity Monitor.app` (and maybe other `top`-like programs). So I should probably back out of that sub-change for now and deal with it later.

**Comment by richardkiss** [comment]
I've replaced the `start_` binaries, so this has at least a chance of generating working `pyinstaller` installations now.

**Comment by richardkiss** [comment]
This is ready for review & merge.

File: src/pyinstaller.spec [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+
+# keyring uses entrypoints to read keyring.backends from metadata file entry_points.txt.
+keyring_datas = copy_metadata("keyring")[0]
+
+version_data = copy_metadata(get_distribution("chia-blockchain"))[0]
+
+block_cipher = None
+
+other = [
+    "aiter.active_aiter",
```
Comment (mariano54) [inline src/pyinstaller.spec:65]: Aiter is not used so we should remove this

**Review by hoffmang9 — APPROVED** [review]
looks good

**Pull request merged**